src/network/supernode1.md

```diff
@@ -0,0 +1,71 @@
+# Supernode 1
+
+![image](images/supernode1-view.jpg)
+
+Supernode 1 is located on top of 200 Woolner Ave, Toronto. The supernode was deployed along side a Cisco deployed infrastructure belonging to the City of Toronto.
+
+## Network
+
+The supernode consists of 2 antennas and a router. The router is connected to the Cisco switch which is connected to the Bell 2000 modem that offers internet access for use as a gateway path.
+
+The router acts as a Babel routing device. The antennas are configured in bridge mode.
+
+![image](images/supernode1-network-layout.png)
+
+## Hardware
+
+### Antenna 1
+
+
+Antenna 1 is an Ubiquiti LAP-120 mounted on the West arm of the building, on the south side of the roof, colocated with Cisco on their mounts. The antenna is facing south.
+
+![image](images/supernode1-hardware-antenna1.jpg)
+
+### Antenna 2
+
+Antenna 2 is an Ubiquiti LAP-120 mounted on the south arm of the building, on the east side of the roof, colocated with Cisco on their mounts. The antenna is facing east.
+
+![image](images/supernode1-hardware-antenna2.jpg)
+
+### Router 1
+
+
+Router 1 is a Ubiquti EdgeRouter X-SFP mounted inside the Cisco cabinet in the ballast room. It is connected to the exit node over a L2TP tunnel using UDP.
+
+Router is configured with a static IP and only routes to the exit node and a secondary VPS server. The secondary VPS is used to provide an OpenVPN tunnel for secure remote management of the device.
```

Comment:
Explain what is the `exit node`.

Reply:
Exit node will have its own page anyway. 

As for the VPS I'm not going to document this. Its a back door for in a temporary solution in case L2TP collapses in a unfinished exit node configuration.